CONTRIBUTING.md

```diff
@@ -1,23 +1,57 @@
 # Contributing
 
-This document provides guidelines for contributing to the module.
+This document provides guidelines for contributing to the CloudBees CD/RO add-on for Amazon EKS blueprints.
 
 ## Dependencies
 
-Validate your changes inside the blueprint-agent described in [.Dockerfile](blueprints/Dockerfile). It can be run `make dBuildAndRun`.
+Validate your changes inside the blueprint agent, as described in [Dockerfile](blueprints/Dockerfile) For example, it can be used to run `make dBuildAndRun`.
 
-## Pre-commits: Linting, Formatting and Secrets Scanning
+<!---
```

Comment:
I added the following sections, but commented them out for now, so we can simply remove the comments once we're ready to use them.